readme.md

```diff
@@ -28,11 +28,11 @@ decompress('unicorn.tar.gz', 'dist', {
 
 ## API
 
-### decompressTarbz()(input)
+### decompressTarbz()(buf)
```

Comment:
Revert the name to `input` and add an extra `options` parameter

`decompressTarbz()(input, [options])`

Also document the new `options` object by pointing to the `options` parameter of `decompress-tar`. The last one has to wait untill the other PR is merged though.